bin/pip_find_builddeps.py

```diff
@@ -96,18 +132,17 @@ def main():
     )
 
     args = ap.parse_args()
-    builddeps = find_builddeps(args.requirements_files, no_cache=args.no_cache)
 
-    # Month Day Year HH:MM:SS
-    date = datetime.datetime.now().strftime("%b %d %Y %H:%M:%S")
+    log.info(
+        "Please make sure the input files meet the requirements of this script "
+        "(see --help)"
+    )
 
-    lines = [f"# Generated by {SCRIPT_NAME} on {date}"]
-    if builddeps:
-        lines.extend(builddeps)
-    else:
-        lines.append("# <no build dependencies found>")
+    builddeps = find_builddeps(args.requirements_files, no_cache=args.no_cache)
+    file_content = generate_file_content(builddeps)
 
-    file_content = "\n".join(lines)
+    log.info("Will write output to %s", args.output_file or "stdout")
```

Comment:
Is this one needed as `info`? The command input/output should say this already (if an output file is given, it will be in the user command, otherwise, the output will be shown in the screen).